week9_homework/week8_homework/displayTodos.js

```diff
@@ -0,0 +1,62 @@
+// Homework todo server
+// 1.	Sa afisam o lista de TODO items
+import {getTodos} from './api.js'
+import './addTodo.js'
+import createDeleteButton from './removeTodo.js'
+import createCheckButton from './updateTodo.js'
+
+getTodos().then(function(todos){
+    const boxTodosHtml = document.getElementById("todo-box");
+    const todoListHtml = document.getElementById("todo-list");
+
+    //Cum ar trebui sa fac aici pentru a folosi .forEach() ?
+    // todos.forEach((todos, i){
```

Comment:
```suggestion
     // todos.forEach((todos, i) => {
```
aici trebui arrow function sau function normal ca sa merga